front-end/App.js

```diff
@@ -23,7 +24,6 @@ export default function App() {
       // eslint-disable-next-line no-console
       .catch(err => console.error(err))
   }, [])
-
   return isReady ? (
     <Container style={style.container}>
       <LoanNav />
```

Comment:
@JialinWang1 why do you have another `Loan` component with the same exact content?